shell/packages/accounts-email-token/token_common.js

```diff
@@ -0,0 +1,44 @@
+Accounts._hashToken = function (token) {
+  return {
+    digest: SHA256(token),
+    algorithm: "sha-256"
+  };
+};
+
+var enabled = false;
+
+Meteor.enableEmailTokenLogin = function () {
```

Comment:
Any particular reason these methods are going on `Meteor`? If so, comment. If not, maybe they should go in `Accounts` or some object specific to this package.